LICENSES/README

```diff
@@ -0,0 +1,60 @@
+The systemd project uses a single line reference to Unique License Identifiers
+in source files as defined by the Linux Foundation's SPDX project
+(https://spdx.org/).
+
+The current set of valid, predefined SPDX identifiers can be found on the SPDX
+License List at https://spdx.org/licenses/.
+
+The 'license/' directory contains all the licenses used by the sources included
+in the systemd project source tree.
+
+Unless otherwise noted, the systemd project sources are licensed under the terms
+and conditions of the "GNU Lesser General Public License v2.1 or later".
+
+The following exceptions apply:
+
+ * the udev sources are licensed under GPL-2.0-or-later
+ * the header files contained in src/basic/linux/ and src/shared/linux/ are
+   copied verbatim from the Linux kernel source tree and are licensed under
+   GPL-2.0 WITH Linux-syscall-note and are used within the scope of the
+   Linux-syscall-note exception, and thus their license does not affect the rest
+   of the source code/binaries
+ * the src/shared/initreq.h header is licensed under original license,
+   LGPL-2.0-or-later.
+ * the src/shared/linux/bpf_insn.h header is copied from the Linux kernel
+   source tree and is licensed under either BSD-2-Clause or GPL-2.0-only,
+   and thus is included in the systemd build under the BSD-2-Clause license.
+ * The src/basic/linux/wireguard.h header is copied from the Linux kernel
+   source tree and is licensed under either MIT or GPL-2.0 WITH
+   Linux-syscall-note, and thus is included in the systemd build under the MIT
+   license.
+ * the following sources are licensed under the MIT license:
+   - hwdb.d/parse_hwdb.py
+   - man/glib-event-glue.c
+   - src/basic/linux/batman_adv.h
+   - src/basic/sparse-endian.h
+   - tools/catalog-report.py
+ * the following sources are licensed under the CC0-1.0 license:
+   - src/basic/siphash24.c
+   - src/basic/siphash24.h
+   - src/systemctl/systemd-sysv-install.SKELETON
+   - tools/check-includes.pl
+ * the following sources are licensed under the LicenseRef-murmurhash2-public-domain:
+   - src/basic/MurmurHash2.c
+   - src/basic/MurmurHash2.h
+ * the following sources are licensed under the LicenseRef-lookup3-public-domain:
+   - src/libsystemd/sd-journal/lookup3.c
+   - src/libsystemd/sd-journal/lookup3.h
+
+OpenSSL notes
```

Comment:
maybe convert the whole file into markdown style? it's close enough already... i.e. this line should use `#`